apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/styles/starter-page-templates-editor.scss

```diff
@@ -69,7 +69,26 @@ body.admin-bar:not( .is-fullscreen-mode ) .page-template-modal-screen-overlay {
 
 // Close button
 .page-template-modal .components-modal__header .components-button {
-	left: 0;
+	left: -24px; // Make up for 24px modal padding.
```

Comment:
I think this value needs to be made into a sass variable. It's littered around the CSS (my fault!) and needs encapsulating.